spock-specs/src/test/groovy/org/spockframework/smoke/extension/ParallelSpec.groovy

```diff
@@ -0,0 +1,34 @@
+package org.spockframework.smoke.extension
+
+import org.spockframework.EmbeddedSpecification
+
+import java.util.concurrent.CountDownLatch
+import java.util.concurrent.atomic.AtomicInteger
+
+import static java.util.concurrent.TimeUnit.MILLISECONDS
+
+class ParallelSpec extends EmbeddedSpecification {
+
+  static void incrementBlockAndCheck(AtomicInteger sharedResource, CountDownLatch countDownLatch)
+    throws InterruptedException {
+    int value = incrementAndBlock(sharedResource, countDownLatch)
```

Comment:
Having `throws` with the same indentation as the first code line reduces readiness (x3).